docs/en/advanced_guides/models.md

```diff
@@ -0,0 +1,330 @@
+# Design Your Own Models
+
+MMGeneration is built upon MMEngine and MMCV, which enables users to design new models quickly, train and evaluate them easily. In this section, you will learn how to design your own models.
+
+## Overview of models in MMGeneration
+
+In MMGeneration, one algorithm can be splited two compents: **Model** and **Architecture**(s).
+
+- **Model** are topmost wrappers and always inherint from `BaseModel` provided in MMEngine. **Model** is responsible to network forward, loss calculation and backward, parameters updating, etc. In MMGeneration, **Model** should be registered as `MODELS`.
+- **Architecture**(s) is the neural network to train or inference, and always as element(s) of **Model**. In MMGeneration, **Architecture** should be registered as **MODULES**.
+
+Take BigGAN model as an example, [generator and discriminator](https://github.com/open-mmlab/mmgeneration/blob/test-1.x/mmgen/models/architectures/biggan/generator_discriminator.py), [`BigGAN`](https://github.com/open-mmlab/mmgeneration/blob/test-1.x/mmgen/models/gans/biggan.py) is the **Model**, which take data from dataloader and train generator and discriminator alternatively.
+
+You can find implementation **Model** for [GAN](https://github.com/open-mmlab/mmgeneration/tree/test-1.x/mmgen/models/gans) and [Diffusion](https://github.com/open-mmlab/mmgeneration/tree/test-1.x/mmgen/models/architectures) models here. And implementation of **Architecture**(s) can be find [here](https://github.com/open-mmlab/mmgeneration/tree/test-1.x/mmgen/models/architectures).
```

Comment:
```suggestion
You can find implementation **Model** for [GAN](https://github.com/open-mmlab/mmgeneration/tree/dev-1.x/mmgen/models/gans) and [Diffusion](https://github.com/open-mmlab/mmgeneration/tree/dev-1.x/mmgen/models/diffusion) models here. And implementation of **Architecture**(s) can be find [here](https://github.com/open-mmlab/mmgeneration/tree/dev-1.x/mmgen/models/architectures).
```

Reply:
Actually we offer ``gans``, ``translation_models``, ``diffusions``, ``losses``, ``architectures`` , these should all be covered.

Reply:
You can directly give a list like these.
Model
   xxx()
Module
   xxx()
Denote we should mention that losses are regarded as MODULES im MMGeneration.